src/content/plugins/split-chunks-plugin.md

```diff
@@ -209,11 +209,11 @@ The difference between `maxInitialSize` and `maxSize` is that `maxInitialSize` w
 
 ### `splitChunks.name`
 
-`boolean = true` `function (module, chunks, cacheGroupKey) => string` `string`
+`boolean = false` `function (module, chunks, cacheGroupKey) => string` `string`
 
 Also available for each cacheGroup: `splitChunks.cacheGroups.{cacheGroup}.name`.
 
-The name of the split chunk. Providing `true` will automatically generate a name based on chunks and cache group key.
+The name of the split chunk. Providing `false` will keep the same name of the chunks so it doesn't change names unnecessarily.
```

Comment:
Line 224
> T> It is recommended to set `splitChunks.name` to `false` for production builds so that it doesn't change names unnecessarily.

I think this tip is not needed anymore then.

Reply:
@snitin315  I think it's still necessary to show that it's the recommended option for production.

Reply:
How about merging both sentences?